Examples/moltin iOS Example/Models/ProductCategory.swift

```diff
@@ -13,8 +13,8 @@ class ProductCategory: moltin.Category {
     var backgroundImage: String?
 
     enum ProductCategoryCodingKeys: String, CodingKey {
-        case backgroundColor = "background_colour"
-        case backgroundImage = "background_image"
+        case backgroundColor = "backgroundColor"
+        case backgroundImage = "backgroundImage"
```

Comment:
If the key and value are the case, you don't need to define the key, so this can just be 
```suggestion
        case backgroundImage
```